### ISSUE TITLE
When I enter any links, it'll stay in the queue without ever downloading

### ISSUE THREAD

**Issue by TheIceMagmaCube** [AllToMP3/alltomp3-app#238]
Hi !
I downloaded AllToMP3 and tried to use it. Sadly, it never downloaded any songs.
I'm using the last version and Windows 10 Home (21H2)

For testing I used this link. I tried some more on YouTube but it doesn't work for an unknow reason.
https://open.spotify.com/album/6hPkbAV3ZXpGZBGUvL6jVM?si=JC9APOr7RgOTXXlN3a1VeA

![image_2022-04-18_104130769](https://user-images.githubusercontent.com/38074756/163782245-22fea010-458d-4a4e-bd49-cce14da6430b.png)

I disabled my firewall, antivirus and launched the program with administrator rights, and try to reinstall it but non of these attemps was sucessful.

More, I tried to change the download localisation, but it didn't help.

Thanks for your hard work!

**Comment by honzahk** [comment]
Hello, having the exact same issue under exact same circumstances

**Comment by honzahk** [comment]
Hello, having the exact same issue under the exact same circumstances

**Comment by recoilcontrolnone** [comment]
im having the same issue. no clue how to fix

**Comment by haotienwang8** [comment]
same issue on macos

**Comment by personmangirl** [comment]
Same issue for me. entered the spotify playlist and it's just sitting there not starting.

**Comment by adamane** [comment]
Windows 10 Pro 21H2 has the same issue

**Comment by kodymoodley** [comment]
Same on MacOS Monterey 12.3.1

**Comment by LovelyBloom** [comment]
same here.

**Comment by GsHeri** [comment]
same here, Win10

looks like this thing is broken and nobody cares, huh?

**Comment by LovelyBloom** [comment]
obviously the project is abandoned.
go to filecr.com install the extension to become a premium user
install: TubeCable Spotify Downloader
FIXED

**Comment by evokeego** [comment]
Same MacOS Monterey 12.2

**Comment by keego** [comment]
Same MacOS Monterey 12.2

**Comment by keego** [comment]
After digging through the code and running things locally (which was a journey) it looks like it pulls the metadata from Spotify but then searches for the song on YouTube to download it, and in the case of all spotify songs I've tried, it can't find the song on YouTube and silently fails

**Comment by LovelyBloom** [comment]
yeah that's called the "youtube download" method. other apps constantly open your Spotify and request the download from there.
on the note here: even with premium apps youtube download method doesn't work anymore
that's where the problem is. it was never designed to download directly from spofiy

**Comment by GsHeri** [comment]
whuat?? so this is just a youtube downloader???
i'd almost call that a scam

**Comment by LovelyBloom** [comment]
no actually they collaborate. you see your almost all of your Spotify music is on youtube vice-versa. It's how it works.
anyway I found a better one and yes it is technically a youtube downloader called [spotdl ](https://github.com/spotDL/spotify-downloader). it's fast and efficient and so far haven't led me down.

**Comment by keego** [comment]
Oh that looks fire, I'ma have to check that out. Thanks for the rec 👍
